pkg/types/git_project.go

```diff
@@ -24,8 +25,10 @@ func (gp *GitProject) UnmarshalYAML(unmarshal func(interface{}) error) error {
 
 func ValidateGitProject(sl validator.StructLevel) {
 	gp := sl.Current().Interface().(GitProject)
-	if strings.Index(gp.SubDir, "/") != -1 {
-		sl.ReportError(gp.SubDir, "subDir", "SubDir", "/ is not allowed in subDir", "")
+	for _, e := range filepath.SplitList(gp.SubDir) {
```

Comment:
`filepath.SplitList` splits by `:` and `;`. What you want to do is a plain `strings.Split(gp.SubDir, "/")`